### PR TITLE
Naming refactor, added some comments then fixed merge conflicts

### LINKED ISSUE #117: Minor refactor- variable names
**Is your feature request related to a problem? Please describe.**
Some variable names and function don't have good names.

**Describe the solution you'd like**
Rename.

### PR DESCRIPTION
**Pull request by connielion**
## Changes
1. Renamed some functions and variables.
2. Ran prettier.
3. Pulled new styling changes from development.

## Purpose
Rename variables and added some doc comments.

Closes #117

### CODE REVIEW

File: src/components/PokeApp.js [hunk 1]
```diff
@@ -6,14 +6,14 @@ import { fetchPokemon, fetchPokemons } from "../util/fetchPokemonData";
 import "../styles/pokeAppStyle.css";
 import "../styles/main.scss";
 
-function PokeApp() {
+const PokeApp = () => {
```
Comment (BlackCubes) [inline src/components/PokeApp.js:9]: Nice ES6 :O

**Review by BlackCubes — APPROVED** [review]
Clean!

File: src/components/PokeApp.js [hunk 2]
```diff
@@ -6,14 +6,14 @@ import { fetchPokemon, fetchPokemons } from "../util/fetchPokemonData";
...
   // Store search bar input value: could be dex number or pokemon name
   const [pokemonName, setPokemonName] = useState("");
-  // fetchedData is a single Pokemon object the user searched for
-  const [fetchedData, setFetchedData] = useState([]);
+  // singlePokemon is a single Pokemon object the user searched for
+  const [singlePokemon, setSinglePokemon] = useState([]);
```
Comment (kmausolf) [inline src/components/PokeApp.js:13]: The day has finally come!

Comment (kmausolf) [reply src/components/PokeApp.js:13]: Although, we should probably refactor this to be an object instead of an array.

**Pull request merged**